src/providers/aws/infrastructure/handlers/ec2_fleet_handler.py

```diff
@@ -404,7 +412,9 @@ def _create_fleet_config_legacy(
                 }
             ],
             "TargetCapacitySpecification": {"TotalTargetCapacity": request.requested_count},
-            "Type": template.fleet_type,
+            "Type": template.fleet_type.value
```

Comment:
I think we should try to use enums everywhere instead of values/str if possible.

Reply:
We probably should use one format after all, but lets worry about it later.